cplusplus/VImage.cpp

```diff
@@ -767,8 +767,9 @@ std::vector<VImage>
 VImage::bandsplit( VOption *options ) const
 {
 	std::vector<VImage> b;
+	b.reserve(bands());
 
-	for( int i = 0; i < bands(); i++ )
+	for( int i = 0; i < int(b.capacity()); i++ )
```

Comment:
I think this line needs to stay as-is. The call to `reserve()` might over-allocate so the vector capacity will not necessarily equal the number of bands.

Reply:
Changed.